### ISSUE TITLE
Accessibility -> Enter Data main page does not have a heading structure

### ISSUE THREAD

**Issue by kgorana** [bcgov/bcparks-ar-api#257]
## Description
**How to replicate:**
* Navigate to the A&R on Test environment and log in: [A&R Admin](https://d3ahrzu0mpndj3.cloudfront.net/enter-data)
* Navigate to the Enter Data page and run the Wave accessibility tool
* Check for errors

**Expected Behavior:**
* Provide a clear, consistent heading structure, generally one <h1> and sub-headings as appropriate. Except for very simple pages, most web pages should have a heading structure

**Issue:**
* The Enter Data page does not have a heading structure. This is affecting the users who use assistive technology

## Linked issues
relates to
- [x] ~[BRS-822](https://bcparksdigital.atlassian.net/browse/BRS-822)~
